streamly-examples.cabal

```diff
@@ -59,6 +64,12 @@ common exe-dependencies
     , vector                >= 0.12  && < 0.13
     , mtl                   >= 2.2   && < 3
 
+  if flag(interop)
+    build-depends:
+        pipes                 >= 4.3.16 && < 4.4
+      , streaming             >= 0.2.3.0 && < 0.3
+      , conduit               >= 1.3.4.1 && < 1.3.5
```

Comment:
We can add these as individual dependencies for each executable. If someone wants to build only one interop  example they should not be bringing all of them.